Remove gender-specific reference

andynava opened the pull request:
No need to specify a gender ("guys").

"Microsoft staff", which is gender neutral, is mentioned in the linked article (https://docs.microsoft.com/en-us/azure/application-insights/app-insights-data-retention-privacy), so I recommend using that.

azurecla commented:
Hi __@adhurwit__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

BryanLa commented:
Merging per conversation w/Carol Bailey.

andynava committed:
Remove gender-specific reference

No need to specify a gender ("guys")

In the linked article (https://docs.microsoft.com/en-us/azure/application-insights/app-insights-data-retention-privacy), "Microsoft staff", which is gender neutral, is referenced, so I recommend using that.

GitHubber17 committed:
Update app-insights-proactive-performance-diagnostics.md
